### DISCUSSION TITLE
The MySQL client still needs to be installed to import the. SQL file

### DISCUSSION THREAD

**Discussion by iseesec** [dbeaver/dbeaver#11135] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: macos big sur 11.1
- DBeaver version：版本7.3.0.202011291229

#### Connection specification:
- Database name and version: 8.0.19

#### Describe your question:The MySQL client still needs to be installed to import the. SQL file

Why do I still need to install MySQL client to import. SQL files? I don't need to use dbeaver to install MySQL client. Moreover, Navicat can import directly without installing the client.

It is recommended to be the same as the built-in jdbc driver

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please try  the latest release 7.3.1 https://dbeaver.io/download/. Native client was added to Mac https://github.com/dbeaver/dbeaver/issues/7976
